packages/database/src/migrations/20200915040216-FixSamoaEntities-modifies-data.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+/**
+  * This migration deletes two districts from the database
+  * and updates one existing facility by fixing/adding it in relevant locations
+  */
+
+//Below are the ids of all the entities being updated in various tables
+
+ const newFacilityEntityId = generateId();
+ const motootuaDistrictEntityId = '5eba39e161f76a3da3000011';
+ const tuamasagaDistrictEntityId = '5eba39e161f76a3da3000010';
+ const upoluGeoAreaId = '5df1b88c61f76a485cd65ea6';
+ const motootuaHospitalClinicId = '5a7bda413ec0d460d2b06d08';
+ const motootuaGeoAreaId = '5a7bda413ec0d460d2ac6024';
+ const tuamasagaGeoAreaId = '5a7bda413ec0d460d2a77ef8';
+
+//This is a really dumb solution to solve an issue where an existing district that will be deleted has the code of a facility being added to the entity table
+ const tempCode = 'thisCodeIsTemp';
+
+exports.up = function(db) {
+  return db.runSql(`
+
+    update entity set code='${tempCode}' where id='${motootuaDistrictEntityId}';
+
+    INSERT INTO "entity" ("id", "code", "parent_id", "name", "type", "point", "region", "image_url", "country_code", "bounds", "metadata", "attributes")
+    VALUES 
+    (
+      '${newFacilityEntityId}',
+      'WS_SAMOA_1',
+      '5df1b88c61f76a485cd7815a',
+      'Moto''otua Hospital',
+      'facility',
+      NULL,
+      NULL,
+      NULL,
+      'WS',
+      NULL,
+      '{"dhis": {"isDataRegional": true}}',
+      '{}'
+    );
+
+    update survey_response set entity_id='${newFacilityEntityId}' where entity_id='${motootuaDistrictEntityId}';
+
+    delete from entity where id='${motootuaDistrictEntityId}';
```

Comment:
```suggestion
    delete from entity where id in ('${motootuaDistrictEntityId}', '${tuamasagaDistrictEntityId}');
```
And can delete the line below. Similar suggestion for the 2x geo area lines